string id
float64[] dimensions
geometry_msgs/Pose corner_frame
geometry_msgs/Pose center_frame

